tests/test_barostat_water_density.py

```diff
@@ -36,17 +35,13 @@
 
     # generate an alchemical system of a waterbox + alchemical ligand:
     # effectively discard ligands by running in AbsoluteFreeEnergy mode at lambda = 1.0
-    mol_a, _, core, ff = hif2a_ligand_pair.mol_a, hif2a_ligand_pair.mol_b, hif2a_ligand_pair.core, hif2a_ligand_pair.ff
-    complex_system, complex_coords, complex_box, complex_top = build_water_system(
-        initial_waterbox_width.value_in_unit(unit.nanometer)
-    )
-
-    min_complex_coords = minimize_host_4d([mol_a], complex_system, complex_coords, ff, complex_box)
-    afe = AbsoluteFreeEnergy(mol_a, ff)
-
-    unbound_potentials, sys_params, masses, coords = afe.prepare_host_edge(
-        ff.get_ordered_params(), complex_system, min_complex_coords
+    mol_a, ff = (
+        hif2a_ligand_pair.mol_a,
+        hif2a_ligand_pair.ff,
```

Comment:
Formatting quirk: Beware that the trailing comma is significant in the eyes of the black formatter: can toggle whether you want to put everything on one line (probably preferred here)  or split each list element on a separate line: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma